wavpackdll/wavpackdll.rc

```diff
@@ -34,7 +34,7 @@ END
 
 2 TEXTINCLUDE 
 BEGIN
-    "#include ""afxres.h""\r\n"
+    "<windows.h>\r\n"

```

Comment:
`winresrc.h` wolud be better

Reply:
And it should be with the `#include ` directive

Reply:
Yes of course, oopsy :)

Reply:
Fixed in e8ac75f9ff9d4564117b6204323f3ed4845e910b